src/pages/unAuth/indexOld.js

```diff
@@ -0,0 +1,568 @@
+/* This example requires Tailwind CSS v3.0+ */
+import { useState, useEffect, useRef, useCallback } from 'react';
+import { Header } from '../../components/common/header';
```

Comment:
I don't understand the file name, what is `indexOld`.?
if we don't need it let's delete it then